### ISSUE TITLE
Talk to us and to eachother

### ISSUE THREAD

**Issue by jpmens** [owntracks/talk#1]
This "issue" tracker is where you can ask questions and get help from the OwnTracks community. Tell us and others what you do with OwnTracks, which apps you use, and whether you've built something neat. If you need help, open an issue here and cross your fingers that somebody will, indeed help you. 😃

**Comment by miknuc** [comment]
Hi there!
This forum seems to be a bit empty yet, let's change this :-)

I am currently exploring owntracks. I installed the Android app, got a MQTT server on my raspberry that serves as external server and forwarded the MQTT messages to my interal fhem server. Works. Thanks for the good work!

Next step I tried to track & display my routes. Therefore 

1. I planned to install ot-recorder on my raspberry. Unfortunately, this is still a wheezy system. You provide a debian imgage for this, but this image depends on libsodium and this lib is not part of wheezy. I decided against building libsodium from scratch :(

2. I considered to use the docker image but I find the documentation in the booklet a bit on the short side:
- the example states to use  addresses 192.168.1.82 and **10.0.1.83** but the docker command lists 
`-e IPLIST="192.168.1.1 192.168.1.82" ` 
- my server does not use 2 ip-adresses. May I just omit one?
- launching docker fails:
`docker run -v /var/owntracks:/owntracks -p 11883:1883 -p 18883:8883 -p 8083:8083 \
    --name owntracks-recorder  
    --hostname ds415.ww.mens.de \
    -e MQTTHOSTNAME="ds415.ww.mens.de" \
    -e IPLIST="192.168.1.1 192.168.1.82" \
    -e HOSTLIST="mqtt.ww.mens.de ds415.ww.mens.de" \
    owntracks/recorderd` 
(there seems to be a \ missing after --name owntracks-recorder )
- I am new to docker. My impression is that the variables "MQTTHOSTNAME" etc. are defined in the container. Is there a documentation which variables are available?
